prisma/migrations/20210831100246_initial_login_schema/migration.sql

```diff
@@ -0,0 +1,38 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "givenName" TEXT NOT NULL,
+    "familyName" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "UserRole" (
+    "userId" INTEGER NOT NULL,
+    "roleId" INTEGER NOT NULL,
+
+    PRIMARY KEY ("userId","roleId")
+);
+
+-- CreateTable
+CREATE TABLE "Role" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "User.email_unique" ON "User"("email");
```

Comment:
why would a sequence number be a vulnerability? Even if it can be predicted. How would that be a vulnerability. 

Having a separate primary ID, allows for decoupling from the email. Maybe even allow multiple emails in the future.

Reply:
What I will do is change this go `uuid`
https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#uuid

Reply:
Changed to uuid:
https://github.com/bloomberg/coffeechat/pull/20/commits/e2fcd419823241b93035bc170ade830d6f44e2e3